### DISCUSSION TITLE
Budgets, categories and tag organization

### DISCUSSION THREAD

**Discussion by wazabees** [firefly-iii/firefly-iii#8211] [General]
I'm looking for a bit of inspiration when it comes to organizing transactions in Firefly. I've read about [organizing transactions in the documentation](https://docs.firefly-iii.org/firefly-iii/financial-concepts/organizing/) (btw, it says you can still set an amount available to budget, which I think has changed now) and I'm still not sure what makes sense.

I buy music related stuff. It can be a digital download from Bandcamp, a vinyl record, a CD or a streaming subscription. Subscriptions are recurring, so that'll be in a budget, but everything else is incidental and wont' have a budget.

Would you use a category called "Music" for these different things, and tags to specify "vinyl records or "digital download"? Or would you spread it over multiple categories, for "vinyl records", digital download" and just use a tag for "Music"? Or am I being far too granular?

One thing that comes to mind, is that I can click on a tag, and see a list of which categories the different transactions have assigned, but I can't click on a category and see which tags the transactions have, so maybe using a tag for "Music" is the better option. Or maybe I'm thinking about it the wrong way.

Thanks ;)

**Comment by JC5** [comment]
I tend to use tags, but too much granularity doesn't add much value to me. It's perhaps personal :)
